localtime.go

```diff
@@ -44,15 +44,17 @@ type LocalDate struct {
 func LocalDateOf(t time.Time) LocalDate {
 	var d LocalDate
 	d.Year, d.Month, d.Day = t.Date()
+
```

Comment:
Not found of those extra new lines.

Reply:
it's the wsl linter. We can deactivate if you prefer
I think the default confs usually make sense, I broke many of my habits to fit them and honestly got use to it, it makes things kinda easier to read. It's your choice though.

From the wsl README:

WSL is a linter that enforces a very non scientific vision of how to make code more readable by enforcing empty lines at the right places.

Reply:
Looking at the wsl configuration, I would set `allow-assign-and-anything: true`.